lib/commands/project/create.ts

```diff
@@ -0,0 +1,70 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+import ProjectCommandBaseLib = require("./project-command-base");
+import path = require("path");
+import options = require("./../../options");
+
+export class CreateCommand extends ProjectCommandBaseLib.ProjectCommandBase {
+	private static GENERATOR_NAME = "generator-kendo-ui-mobile";
+
+	constructor(public $project: Project.IProject,
+				public $errors: IErrors,
+				public $logger: ILogger,
+		private $childProcess: IChildProcess,
+		private $projectConstants: Project.IProjectConstants,
+		private $appScaffoldingExtensionsService: IAppScaffoldingExtensionsService,
+		private $screenBuilderService: IScreenBuilderService) {
+		super($project, $errors, $logger);
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			try {
+				this.$screenBuilderService.prepareScreenBuilder(CreateCommand.GENERATOR_NAME).wait();
+				var appScaffoldingPath = this.$appScaffoldingExtensionsService.appScaffoldingPath;
+				var projectDirPath = path.resolve(options.path || ".");
```

Comment:
Currently we are creating files in the current dir. This will be fixed after this PR with changes on the ScreenBuilder side.